src/test/java/seedu/canoe/logic/parser/AddStudentToTrainingCommandParserTest.java

```diff
@@ -0,0 +1,42 @@
+package seedu.canoe.logic.parser;
+
+import static seedu.canoe.logic.commands.CommandTestUtil.VALID_ID_LIST_2;
+import static seedu.canoe.logic.commands.CommandTestUtil.VALID_ID_LIST_6;
+import static seedu.canoe.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.canoe.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.canoe.testutil.TypicalIndexes.INDEX_FIRST_TRAINING;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.canoe.logic.commands.AddStudentToTrainingCommand;
+import seedu.canoe.model.student.Id;
+
+public class AddStudentToTrainingCommandParserTest {
```

Comment:
`AddStudentCommandParserTest` instead since our actual parser class name is `AddStudentCommand`